tests/e2e/operator.sh

```diff
@@ -247,8 +250,8 @@ uninstall_operator() {
 	cmd+="&& ! echo \$_OUT | grep -q -e cc-operator-controller-manager"
 	if ! wait_for_process 180 30 "$cmd"; then
 		echo "::error:: the controller manager is still running"
-
-		local pod_id="$(get_pods_regex $pod $op_ns)"
+		local pod_id
+		pod_id="$(get_pods_regex '$pod' '$op_ns')"
```

Comment:
the same here, double quotes please

Reply:
@ldoktor thanks for the feedback changes applied